lib/lrama/lexer/token.rb

```diff
@@ -1,7 +1,7 @@
 module Lrama
   class Lexer
     class Token < Struct.new(:type, :s_value, :alias, keyword_init: true)
-      Type = Struct.new(:id, :name, keyword_init: true)
+      require 'lrama/lexer/token/type'
```

Comment:
Could you move this "require" to the first line of this file?

Reply:
Done. I updated this PR.